src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCollectionDebuggerProxy.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Text.RegularExpressions
+{
+    internal sealed class RegexCollectionDebuggerProxy<T>
```

Comment:
This is a good question for some folks on the debugger/EE side. @tmat @cston Is there another option to provide these debugger proxies in a more general fashion?